components/gfx/font.rs

```diff
@@ -56,12 +55,11 @@ pub trait FontTableTagConversions {
 
 impl FontTableTagConversions for FontTableTag {
     fn tag_to_str(&self) -> String {
-        unsafe {
-            let pointer = mem::transmute::<&u32, *const u8>(self);
-            let mut bytes = slice::from_raw_parts(pointer, 4).to_vec();
-            bytes.reverse();
-            String::from_utf8_unchecked(bytes)
-        }
+        let bytes = [(self >> 24) as u8,
+                     (self >> 16) as u8,
+                     (self >>  8) as u8,
+                     (self >>  0) as u8];
+        str::from_utf8(&bytes).unwrap().to_owned()
```

Comment:
This is the same way harfbuzz handles tags.  See http://mxr.mozilla.org/mozilla-central/source/gfx/harfbuzz/src/hb-common.h#94 .